docker-compose.client.yml

```diff
@@ -15,20 +15,32 @@ services:
       - BALENACLOUD_APP_NAME=${BALENACLOUD_APP_NAME}
     depends_on:
       - core
+    networks:
+      - levi
```

Comment:
How about? qemu-bridge since this is Qemu specific?

Reply:
Along those lines, maybe `virtual-bridge` to make it less qemu specific. Anyway, all work for me.

Do you want to take this PR over @vipulgupta2048 and make sure it doesn't break anything?

Reply:
Sure thing, I will do a once over and learn more about this.